server/config/options.go

```diff
@@ -180,3 +180,7 @@ type QuotaConfig struct {
 	LimitUpdateInterval       int64 `mapstructure:"limit_update_interval" yaml:"limit_update_interval" json:"limit_update_interval"`
 	TenantSizeRefreshInterval int64 `mapstructure:"tenant_size_refresh_interval" yaml:"tenant_size_refresh_interval" json:"tenant_size_refresh_interval"`
 }
+
+func IsIndexingStoreReadEnabled() bool {
+	return DefaultConfig.Search.WriteEnabled && DefaultConfig.Search.ReadEnabled
```

Comment:
I think `Enabled` and `ReadOnly` options would be easier to understand.

Reply:
So Enabled if false doesn't mean that read is disabled as well it will only disable writes?